Jenkinsfile

```diff
@@ -10,7 +10,7 @@ node {
     def TEST_LEVEL='RunLocalTests'
     def PACKAGE_NAME='0Ho1U000000CaUzSAK'
     def PACKAGE_VERSION
-
+    def SF_INSTANCE_URL = env.SF_INSTANCE_URL ?: "https://login.salesforce.com"
```

Comment:
You don't actually use this on line 41. Can you add it there like in your other PR?